lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -685,7 +684,14 @@ instance ToText (ApiT XPrv) where
         . getApiT
 
 instance FromText (ApiT (Hash "encryption"))  where
-    fromText = fmap ApiT . hashFromText 64
```

Comment:
Why not `hashFromText 100` :thinking: Can the Scrypt output actually be of different sizes ?

Reply:
if i is `100` then we should have `hashFromText 100` but I have this comment : https://github.com/input-output-hk/cardano-wallet/pull/1476#discussion_r397026198